sympy/calculus/util.py

```diff
@@ -1212,11 +1226,16 @@ def __div__(self, other):
                 if S.Zero not in other:
                     return self * AccumBounds(1/other.max, 1/other.min)
 
-                if S.Zero in self and S.Zero in other:
+                if (self.min.is_extended_nonpositive and self.max.is_extended_nonnegative) \
+                    and (other.min.is_extended_nonpositive and other.max.is_extended_nonnegative):
                     if self.min.is_zero and other.min.is_zero:
                         return AccumBounds(0, oo)
                     if self.max.is_zero and other.min.is_zero:
                         return AccumBounds(-oo, 0)
+                    if self.max.is_zero and other.max.is_zero:
+                        return AccumBounds(-oo, 0)
```

Comment:
I think that this should be `Accumbounds(0, oo)`. Can you see, why?

Reply:
Why it should be `AccumBounds(0,oo)` ?

Reply:
For the same reason as in the first case above: if the dividend and divisor are both positive or both negative, then the quotient is positive.

However, on closer inspection it looks like it is best to return `AccumBounds(-oo, oo)` in all of these cases because a sequence can approach zero from either side so its sign cannot be determined from a condition like `self.max.is_zero`. Therefore, remove these special cases and return only `AccumBounds(-oo, oo)`.

Reply:
So if `S.Zero in self and S.Zero in other` it should return `AccumBounds(-oo,oo)` ?

Reply:
Yes, I think so. That is probably the best that can be achieved in general.